src/apps/explorer/components/common/Search/Search.styled.ts

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components'
+import SVG from 'react-inlinesvg'
+
+export const Wrapper = styled.form`
+  display: flex;
+  width: 100%;
+  max-width: 50rem;
+  margin: 0 auto;
+  position: relative;
+
+  > input {
+    height: 5rem;
+    flex: 1 1 auto;
+    background: ${({ theme }): string => theme.greyOpacity};
+    font-weight: ${({ theme }): string => theme.fontMedium};
+    font-size: 1.6rem;
+    border-radius: 0.6rem;
+    line-height: 1;
+    display: flex;
+    outline: 0;
+    appearance: none;
+    align-items: center;
+    color: ${({ theme }): string => theme.greyShade};
+    padding: 1.6rem 1.6rem 1.6rem 5rem;
+    box-sizing: border-box;
+    border: 0.1rem solid transparent;
+    transition: border 0.2s ease-in-out;
+
+    &:focus {
+      border: 0.1rem solid ${({ theme }): string => theme.borderPrimary};
+    }
+
+    &::placeholder {
+      color: inherit;
+      transition: color 0.2s ease-in-out;
+    }
+
+    &:focus::placeholder {
+      color: transparent;
+    }
+  }
```

Comment:
You said you would do another component. Changed your mind?

Reply:
Oh I might have misunderstood you. I thought you meant to separate the styles, but now I read you meant to create a specific styled component for the `input` element?

Reply:
Yes, that's what I meant.
Only asked again because I thought you forgot.
No worries if you rather keep it as is.

Reply:
I think your comment makes sense, especially in terms of readability. I will change it for this PR as it's a rather small change.